src/main/scala/fundamentals/level03/ExceptionExercises.scala

```diff
@@ -105,7 +105,10 @@ object ExceptionExercises {
     *
     * Hint: use filter on List using the getName and getAge functions
     */
-  def validPairs: List[(String, String)] = ???
+  def validPairs: List[(String, String)] =
+    personStringPairs.filter {
+      case (name, age) => ???
+    }
```

Comment:
They are already learnt the pair case match in List. Do we want to add the skeleton here? I felt adding a hint for it is enough. But if we felt that people will struggle with it then may be ok.

Reply:
Add it in I reckon.